pages/common/autojump.md

```diff
@@ -0,0 +1,19 @@
+# autojump
+
+> Automatically jump to the directory you mostly visited.
```

Comment:
Perhaps rephrase to "the directory you visited the most", or "the most visited directory".